joern-cli/frontends/rubysrc2cpg/src/main/antlr4/io/joern/rubysrc2cpg/parser/RubyParser.g4

```diff
@@ -177,6 +177,8 @@ indexingArgumentList
         # operatorExpressionListIndexingArgumentList
     |   operatorExpressionList COMMA splattingArgument
         # operatorExpressionListWithSplattingArgumentIndexingArgumentList
+    |   ((symbol|association) COMMA? NL*)*
+        #symbolOrAssociationIndexList
```

Comment:
Could we make the naming a bit more consistent, ie end in "IndexingArgumentList"